bigml/io.py

```diff
@@ -42,11 +42,13 @@ def __init__(self, filename, dialect=csv.excel,
         self.file_handler = None
         self.reader = None
 
-    def __enter__(self):
-        """Opening files
+    def open_reader(self):
```

Comment:
hmm: don't we need also a `close_reader()` method for cases where `open_reader` is explicitly called?